### DISCUSSION TITLE
Not able remove SAML identity from the organisation, How to remove organisation from personal account?

### DISCUSSION THREAD

**Discussion by nandha2012** [community/community#109307] [Other Feature Feedback, Questions, & Ideas] [answered]
### Body

Tried attempted to log in to the organization's Git profile but ended up in my personal profile. After receiving an invitation from the organization, unable to accept it. When attempting to log in, they encounter the following error.

There was an issue joining the enterprise: Your GitHub user account @sosafe-nandha-kumar is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'x' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance. and External ID 'x' is already linked to @nandha2012's account. 

My personal profile is not listed in the people list of the organization - checked with the admin
need to unlink the personal account from the organization.

**this is very urgent and would appreciate someone to help us**

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nandha2012 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by rittikamckesson** [reply]
Hi Nandha, facing the same issue. Can you guide how did itresolve
?

**Reply by nandha2012** [reply, marked as answer]
Sorry for the late reply
Unfortunately, I couldn't unlink my personal account from the organization. To resolve the issue, I decided to leave it as it was and created a new GitHub account 
If you have the same problem, I suggest considering creating a new account as a possible solution. If you prefer to keep using your existing account, you might need to work closely with your organization's admin or GitHub support to unlink your personal account.
